Servers/UI/OJS.Servers.Ui/ClientApp/src/components/problems/problem-submissions/SubmissionResults.tsx

```diff
@@ -42,24 +44,33 @@ const SubmissionResults = ({ problemId }: ISubmissionResultsProps) => {
         }
     }, [ getSubmissionResults, problemId ]);
 
-    useEffect(() => {
+    const reload = () => {
         (async () => {
             await getResults();
         })();
+    };
+
+    useEffect(() => {
+        if (currentProblemSubmissionResults.length !== 0) { return; }
+        reload();
     }, []);
```

Comment:
Missing dependencies?

Reply:
Yes, because i need this to be executed only on component render and if ```currentProblemSubmissionResults``` data is not already loaded, because the tabs component makes the SubmissionResults compoent rerender and a request to the backend is sent everytime.